lms/views/basic_lti_launch.py

```diff
@@ -131,21 +132,42 @@ def blackboard_copied_basic_lti_launch(self):
         """
         Respond to a launch of a newly-copied Blackboard assignment.
 
-        Find the document_url for resource_link_id_history and make a copy of
-        it with the new resource_link_id, then lanuch the assignment as normal.
+        For more about Blackboard course copy see the BlackboardCopied
+        predicate's docstring.
+        """
+        return self.course_copied_basic_lti_launch(
+            BlackboardCopied.get_original_resource_link_id(self.request)
+        )
+
+    @view_config(brightspace_copied=True)
+    def brightspace_copied_basic_lti_launch(self):
+        """
+        Respond to a launch of a newly-copied Brightspace assignment.
+
+        For more about Brightspace course copy see the BrightspaceCopied
+        predicate's docstring.
+        """
+        return self.course_copied_basic_lti_launch(
+            BrightspaceCopied.get_original_resource_link_id(self.request)
+        )
+
+    def course_copied_basic_lti_launch(self, original_resource_link_id):
+        """
+        Respond to a launch of a newly-copied assignment.
+
+        Find the document_url for the original assignment and make a copy of it
+        with the new resource_link_id, then launch the assignment as normal.
+
+        Helper method for the *_copied_basic_lti_launch() methods above.
 
-        For more about Blackboard course copy and the resource_link_id_history
-        param see the BlackboardCopied predicate's docstring.
+        :param original_resource_link_id: the resource_link_id of the original
+            assignment that this assignment was copied from
```

Comment:
Added a helper method for `blackboard_copied_basic_lti_launch()` and `brightspace_copied_basic_lti_launch()`, since the two methods are exactly the same except for the `param_name`. I prefer having two separate views that call a helper method, rather than a single view that looks for both param names, the code works out simpler.